### DISCUSSION TITLE
It would be great if organization pages and profile pages could be customized

### DISCUSSION THREAD

**Discussion by julie777** [community/community#12414] [Other Feature Feedback, Questions, & Ideas]
When I search for an organization or person and land on their "home" page there isn't any information available about the page owner, etc.

I just discovered another issue. I can't search for a organization and get a link to the org page. Instead searching for "org:pdm-project" shows links to all of the repositories for the org.

1. Please have search show org or profile pages when that is what the search matches.

Currently, the "home" page allows
- customizing of pinned repos.  - good
- It allows searching for repos.  - good
- shows a long list of all the repos at the bottom - unnecessary as there is a tab for repos at the top of the page.
- the overview tab on the home page, which is the default, doesn't provide a place to actually provide an overview

I will use the [PyPA] organization "home" as an example.

I would like a section above the pinned repos to allow providing a description of the org or person, and information about the repos they have created, etc. This is essentially a README for the profile page and can be part of the profile for the account. Or it could be a link to a file in one of the account's repos.

There should be a better way to present repos than pinning 6 and showing the rest in a long list. The repo search has a type selector. That isn't all that helpful.

Maybe instead of pinned, and unpinned with search and a long list, an alternative solution would be to allow the owner of the page to tag repos with some sort of group label. They could add a description for the label.
This would allow showing repo groups with the description for the group. Groups could be expanded to show the contents.
The owner could also provide and ordering for the groups.

The list format with expandable groups of repos would be much more space efficient, make it easier to understand how all the various repos for an organization fit together and generally be a better user experience.

An example of my groups might be
- python tools - these are tools either written in python or for use by python developers
- games - some games that I wrote for the fun of it
- prototypes - these are projects that I am using to test proof of concept and are not generally useful to others
- retired - these projects have been retired from active development either for lack of interest, being superceeded by another, or their original purpose is no longer valid

**Comment by jsoref** [comment]
Do you mean like https://github.com/github/ ? See https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile

For users, the equivalent doc is https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme

**Reply by julie777** [reply]
> Do you mean like https://github.com/github/ ? See https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile
> 
> For users, the equivalent doc is https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme

That is exactly what I meant for adding a description to the org/user profile. Thank you. I must have missed it. That covers the information part nicely.

I went and looked and based on me not knowing about it and never having seen an org using it, I think it is not visible enough.
How about if when I edit my profile page it provides an option to create that repo and lets me edit the readme?

I just used it and when I create the repo with my user it does provide me with the information, but it would be much nicer if edit profile had
a create overview button that jumped to create <user/org> repo.

All the other comments are still valid, and based on the need for a repo for the overview, all of the customizations of the profile page could be implemented in a profile repo "layout" file that allows me to 
- define repo groups with a short description of the group
- list the repos in the group
- choose how to present the group/repo information

This would truly be a great feature for users and orgs with large numbers of repos. 
The current way of pinning six and listing the rest really shows its weakness when an org has more than 100 repos. 
Also, by not having groups and a description there is not an obvious reason why the org pinned those six repos.

Using the pypa org (@gaborbernat ) as an example, it would be so helpful if there were categories such as
- virtualenv
- PEP reference implementations
- documentation
- etc.

**Reply by jsoref** [reply]
I believe that user profile repositories are technically older (they date to July 2020 -- https://aboutmonica.com/blog/how-to-create-a-github-profile-readme/) than org profiles (https://github.com/github/.github/commit/980ed70d9d39d407c90e46ddc036ac9b9fd73ea0), but the user profile feature is still treated as a "secret" -- when you create one, you get winked at by the mascot and stuff.

To me that means they're still thinking through how they want to expose the feature.

The rest of it ... well... You could technically create a table using markdown.

Fwiw, I'm not opposed to new features here. I just think they're really still collecting input before they make more changes in the area. I certainly would, and I'm maintain a one-person project.
